src/main/java/core/basesyntax/service/impl/TransactionReaderServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionReaderServiceImpl implements TransactionReaderService {

```

Comment:
This class violates SRP - we read the file **and** parse read information to java objects
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/4df815ca-d150-4507-8733-a1e4ea8b43d9)